build.bat

```diff
@@ -1,5 +1,7 @@
 @ECHO OFF
 
+echo [   INFO] Build starting at %TIME% %DATE%
```

Comment:
I'm fine with adding it, but it'd be good to have it in both `build.bat` as well as `build` for consistency. Sorry to be that guy, but yeah.